src/Microsoft.Health.Fhir.TemplateManagement/Microsoft.Health.Fhir.TemplateManagement.csproj

```diff
@@ -40,16 +53,29 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
+    
+  <ItemGroup>
+    <None Update="oras-OSX">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
   
   <Target Name="PreBuild" BeforeTargets="PreBuildEvent">
     <Exec Command="mkdir ..\..\bin &amp; tar -zcvf ../../bin/Hl7v2DefaultTemplates.tar.gz -C ../../data/Templates/Hl7v2 ." />
     <Exec Command="tar -zcvf ../../bin/CcdaDefaultTemplates.tar.gz -C ../../data/Templates/Ccda ." />
   </Target>
   
-  <Target Name="DownloadOrasFile" BeforeTargets="Build">
+  <Target Name="DownloadWinOrasFile" BeforeTargets="Build">
     <DownloadFile SourceUrl="$(OrasWinUrl)" DestinationFolder="../../bin">
       <Output TaskParameter="DownloadedFile" ItemName="Content" />
     </DownloadFile>
-    <Exec Command="tar -xvf ..\..\bin\oras_0.8.1_windows_amd64.tar.gz -C ..\..\bin &amp; copy ..\..\bin\oras.exe ." />
+    <Exec Command="tar -xvf ..\..\bin\oras_0.8.1_windows_amd64.tar.gz -C ..\..\bin &amp; $(CopyExecutable) ..\..\bin\oras.exe ." />
+  </Target>
+    
+   <Target Name="DownloadOSXOrasFile" BeforeTargets="Build">
+    <DownloadFile SourceUrl="$(OrasOSXUrl)" DestinationFolder="../../bin">
+      <Output TaskParameter="DownloadedFile" ItemName="Content" />
+    </DownloadFile>
+    <Exec Command="tar -xvf ../../bin/oras_$(OrasVersion)_darwin_amd64.tar.gz -C ../../bin &amp;&amp; $(CopyExecutable) ..\..\bin\oras .\oras-osx" />
```

Comment:
The OSX platform uses slash `/` instead of backslash `\`, what about keeping consistent styles in `<Exec>` section?

Reply:
Fixed it in the new pr #291